aviary/variable_info/variables.py

```diff
@@ -583,11 +594,13 @@ class Mission:
         ALTITUDE = 'altitude'
         ALTITUDE_RATE = 'altitude_rate'
         ALTITUDE_RATE_MAX = 'altitude_rate_max'
+        BATTERY_STATE_OF_CHARGE = 'battery_state_of_charge'
         DENSITY = 'density'
         DISTANCE = 'distance'
         DISTANCE_RATE = 'distance_rate'
         DRAG = 'drag'
         DYNAMIC_PRESSURE = 'dynamic_pressure'
+        ELECTRIC_ENERGY = 'electric_energy'
```

Comment:
Since energy isn't really an instantaneous value, this should probably have a more descriptive name like `total_electrical_energy_used` or something

Reply:
`total_mission_electric_energy`? `electric_energy_consumption`
Just throwing out some more suggestions

Reply:
Changed to `cumulative_electric_energy_used`